api/prd/mappings.utils.ts

```diff
@@ -1,19 +1,10 @@
-import { JURISDICTION_SERVICES_MAPPINGS, SERVICE_COURT_TYPE_MAPPINGS } from './mappings.data';
-
-export function getServicesByJurisdiction(jurisdictionIdArray: string[]): string[] {
-  return jurisdictionIdArray.map((jurisdictionId) => JURISDICTION_SERVICES_MAPPINGS[jurisdictionId])
-    .reduce(concatWithoutDuplicates);
-}
+import { SERVICES_COURT_TYPE_MAPPINGS } from '../locations/data/serviceCourtType.mapping';
 
 export function getCourtTypeIdsByServices(ServicesIdArray: string[]): string[] {
-  return ServicesIdArray.map((serviceId) => SERVICE_COURT_TYPE_MAPPINGS[serviceId])
+  return ServicesIdArray.map((serviceId) => SERVICES_COURT_TYPE_MAPPINGS[serviceId])
```

Comment:
Why have we added an S?

Reply:
We are pointing to a different mapping for this now that is called that instead - I will remove the other mapping